.github/workflows/teste2e.yml

```diff
@@ -1,9 +1,19 @@
 name: Run E2E Test for skype-sender lambda
 on:
   push:
-    branches: master
-  pull_request:
-    branches: master
+    branches:
+      - master
+    paths:
+      - '.github/workflows/**'
+      -  'conf/**'
+      -  'daily_messages/**'
+      -  'dummy_lambda/**'
+      -  'holiday_reminder/**'
+      -  'pto_detector/**'
+      -  'qelo/**'
+      -  'read_task_completion_sheet/**'
```

Comment:
If I asked you to manually test the task completion sheet because I made a change to Skype Sender - would you do it?